src/runtime/recipe/internal/recipe-walker.ts

```diff
@@ -66,8 +66,7 @@ export class RecipeWalker extends Walker<Recipe> {
     }
     if (this.onPotentialSlotConnection) {
       for (const particle of recipe.particles) {
-        for (const [name, slotSpec] of particle.getSlotSpecs()) {
-          if (particle.getSlotConnectionByName(name)) continue;
+        for (const slotSpec of particle.getUnboundSlots()) {
```

Comment:
Will this restrict a particle from rendering to two different slots?

Also does this suggest that `particle.getUnboundSlots` should actually be `particle.getUnboundSlotConnections`? (wordy, but more accurate)

Reply:
This should be an isomorphic change; just moving the filter from here into particle to make the API for slot connections look more like the API for handle connections. I think you're right about the naming.